core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -174,13 +191,14 @@ import scala.compat.java8.FutureConverters._
 
 @InternalApi private[akka] class EventsByTagStage(
     session: TagStageSession,
-    fromOffset: UUID,
+    initialQueryOffset: UUID,
```

Comment:
How does the backtracking (and look for missing for new persistenceId) work with regards to the `fromOffset`? Can it deliver events with earlier offset than the fromOffset? I'm thinking of what semantics we have when a query is restarted from a stored offset. Are duplicates expected? Risk of missed events?

Reply:
We currently never go before the `fromOffset` as i don't think that would be expected. 

The risk of missed events is if there were delayed events and it was restarted before they were caught by a back track.

I don't think we should change it to look from before the `fromOffset` but we can document the scenario and users can choose to start from an earlier offset on restart if they can be idempotent. 

Further reaching would be to have an offset that combined a timeuuid and the tag pid sequence numbers, that way we could go back befor the offset and not re-deliver events.